### DISCUSSION TITLE
建议重复字幕截图采用相似度算法合并去重

### DISCUSSION THREAD

**Discussion by HayashiKaori** [sum1re/caption_ocr_tool#24] [General]
这样就能免去手动合并之苦，还能自动对上时间抽，输出 srt 文件。

**Comment by sum1re** [comment]
设置中有 SSIM 和 PSNR 两种算法可选, 不过受滤镜模组的影响, 去重的效果并不稳定. 如果滤镜模组的配置不合理, 去重算法根本不起作用, 就会导致大量相似的图像需要手动合并.
